v2/blockstore/readwrite_test.go

```diff
@@ -26,7 +28,11 @@ import (
 	"github.com/ipld/go-car/v2/internal/carv1"
 )
 
-var rng = rand.New(rand.NewSource(1413))
+var (
+	rng              = rand.New(rand.NewSource(1413))
+	oneTestBlock     = dag.NewRawNode([]byte("fish")).Block
+	anotherTestBlock = dag.NewRawNode([]byte("barreleye")).Block
```

Comment:
I was going to suggest to use https://pkg.go.dev/github.com/ipfs/go-block-format#NewBlock instead, but then I noticed that uses a cidv0 instead of merkledag's cidv1.

Reply:
Interesting; we probably should use both actually. thanks for pointing this out

Reply:
Added both thanks @mvdan